### PR TITLE
[GPTQ Enhence] Add unpack int8 weight for GPTQ

### PR DESCRIPTION
**Pull request by Zhenzhong1**
## Type of Change

Feature Enhence

## Description

Add int8 unpack weight to support int8 GPTQ

## Expected Behavior & Potential Risk

N/A

## How has this PR been tested?

Manually tests.

## Dependency Change?

N/A

### CODE REVIEW

File: neural_speed/convert/convert_quantized_llama.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ def main(args_in: Optional[List[str]] = None) -> None:
     out_path = args.outfile.as_posix()
     model_path = args.model.as_posix()
 
-    model, config, quantize_config = load_quantized_model(model_path)
+    #model, config, quantize_config = load_quantized_model(model_path)
```
Comment (a32543254) [inline neural_speed/convert/convert_quantized_llama.py:109]: remove unuse code

File: neural_speed/convert/common.py [hunk 2]
```diff
@@ -215,6 +217,51 @@ def unpack_weight(qweight, scales, qzeros, q_config):
...
+    torch.bitwise_and(zeros, (2**bits) - 1, out=zeros)
+
+    if bits == 8:
+        zeros = zeros.to(torch.int8 if sym else torch.uint8)
+
+    zeros = zeros + 1
+    try:
+        zeros = zeros.reshape(scales.shape)
+    except:
+        # zeros and scales have different iteam numbers.
```
Comment (a32543254) [inline neural_speed/convert/common.py:240]: iteam  >  item

File: neural_speed/convert/common.py [hunk 3]
```diff
@@ -215,6 +217,51 @@ def unpack_weight(qweight, scales, qzeros, q_config):
...
+                                       wf.unsqueeze(-1)).to(torch.int16 if bits == 8 else torch.int8)
+    torch.bitwise_and(weight, (2**bits) - 1, out=weight)
+
+    if bits == 8:
+        # due to INC add shift bias for sym
+        if sym:
+            shift_bias = 2**(bits - 1)
+            weight -= shift_bias
+        weight = weight.to(torch.int8 if sym else torch.uint8)
+        # due to INC asym return torch.uint8 but backend request int8,
```
Comment (a32543254) [inline neural_speed/convert/common.py:258]: it's seems not straight forward ? why don't let inc return a int8 for asym ?

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by changwangss — APPROVED** [review]

File: neural_speed/convert/common.py (hunk 2 above)
Comment (Zhenzhong1) [reply neural_speed/convert/common.py]: fixed

File: neural_speed/convert/convert_quantized_llama.py (hunk 1 above)
Comment (Zhenzhong1) [reply neural_speed/convert/convert_quantized_llama.py]: fixed

File: neural_speed/convert/common.py (hunk 3 above)
Comment (changwangss) [reply neural_speed/convert/common.py:258]: Torch usually uses torch.uint8 for asym quantization, please aware the request.@xin3he

**Pull request merged**